src/pages/rental-details/rental-details.ts

```diff
@@ -33,8 +33,11 @@ export class RentalDetailsPage {
   ngOnInit() {
     this.items = this.itemsService.getItems();
 
+    // We need to offset the date by the difference between the user's timezone
+    // and UTC 0 to have an accurate date
     let tomorrow = new Date();
-    tomorrow.setDate((new Date()).getDate() + 1);
+    const offset = tomorrow.getTimezoneOffset() / (60 * 24);
```

Comment:
I think this should be abstracted into either a helper function or a property on the component, since the same offset is used in both `ngOnInit` and `onRent`. It would be more efficient to calculate it once.